packages/service-library/src/servicelib/rabbitmq/_models.py

```diff
@@ -49,3 +36,6 @@ def from_namespace_and_method(
     ) -> "RPCNamespacedMethodName":
         namespaced_method_name = f"{namespace}.{method_name}"
         return parse_obj_as(cls, namespaced_method_name)
+
+
+__all__: tuple[str, ...] = ("RPCNamespace",)
```

Comment:
Q: why not import the `RPCNamespace` from the correct module instead via this "redirection" approach?

Reply:
If you remote this line usually the tooling complains about this

Reply:
Right. So here the problem is as follows:
- all these models were originally defined in `servicelib.rabbitmq/_models.py`
-  `RPCNamespace` is now part of the api models, i.e. need to go to `models_library`
- I moved only `RPCNamespace` to `models_library`.
- In order not to refactor right now (and add more noice to the PR), I exposed `RPCNamespace`

On second thought, sinceyou commented it, i will remoev this and refactor the imports

Reply:
> Q: why not import the `RPCNamespace` from the correct module instead via this "redirection" approach?

@GitHK what do you mean by the "redirection" approach?